bindings/nodejs/examples/client/14_build_foundry_output.ts

```diff
@@ -34,8 +35,11 @@ async function run() {
 
         const foundryOutput = await client.buildFoundryOutput({
             serialNumber: 0,
-            // 10 hex encoded
-            tokenScheme: new SimpleTokenScheme('0xa', '0x0', '0xa'),
```

Comment:
@Dr-Electron , it seems this function doesn't work well with numbers. For example, for 10 it returns 0x3130, which is hex for 12592. Same for all the other numbers, and in the Python code as well. Am I doing something wrong ?

Reply:
For numbers in JS you can use `.toString(16)`to convert to hex, so `console.log((40).toString(16)); // 28` or `console.log(40..toString(16)); // 28`
But still needs the hex prefix `0x` added, so own converter function for numbers might be the way to go and since the normal number in JS is limited to f64 range, but token supply can be u64 max and native tokens even u256 max it should probably use Bigint

Reply:
Can this be resolved?